src/toggle/toggle.tsx

```diff
@@ -304,11 +307,11 @@ export const Toggle = ({
     const renderCompositeSection = () => {
         return (
             compositeSectionChildren && (
-                <CompositeSectionContainer>
+                <div>
```

Comment:
this div does not seem to have any use anymore, unless we want to add a `data-id` to make it target-able too

Reply:
hmm the flex column does lay the items out vertically, but thought it could group the items together

Reply:
I see, ok that's fine then, let's keep it